### PR TITLE
jquery pnotify def file for 2.x

### PR DESCRIPTION
**Pull request by lgrignon**

### CODE REVIEW

**Comment by vvakame** [comment]
@lgrignon thank you for your contributions.
please remove `jquery.pnotify/jquery.pnotify-1.x.d.ts`.

**Comment by lgrignon** [comment]
Thanks for your reply. It should be ok now.

File: jquery.pnotify/jquery.pnotify.d.ts [hunk 1]
```diff
@@ -1,173 +1,177 @@
-// Type definitions for jquery.pnotify 1.3.1
+// Type definitions for jquery.pnotify 2.x
 // Project: https://github.com/sciactive/pnotify
-// Definitions by: David Sichau <https://github.com/DavidSichau>
```
Comment (vvakame) [inline jquery.pnotify/jquery.pnotify.d.ts]: please keep this line.

**Comment by vvakame** [comment]
@DavidSichau could you review this PR?

**Comment by DavidSichau** [comment]
@lgrignon thanks for your contributions.

Just a small remark: please do not change the indentation, as so it is very hard to see what you have actually changed.

@vvakame This PR looks OK for me.

**Comment by lgrignon** [comment]
Everything should be ok now.

**Comment by vvakame** [comment]
@lgrignon thanks mate!
@DavidSichau thanks!

**Pull request merged**